src/services/utility-helpers.js

```diff
@@ -556,10 +563,11 @@ export function autoScroll(currentItem, containerRef, toTop = false) {
  * @param {Object} event keydown event
  * @param {String} id player instance ID in VideoJS
  * @param {Boolean} canvasIsEmpty flag to indicate empty Canvas
- * @returns 
+ * @returns {ENum} result of the triggered hotkey action
```

Comment:
It looks like the return type is a string.
```suggestion
 * @returns {String} result of the triggered hotkey action
```